style.css

```diff
@@ -0,0 +1,403 @@
+body {
+  font-family: "Roboto Condensed", sans-serif;
+  height: 100%;
+  width: 100%;
+}
+
+.wrapper {
+  max-width: 1310px;
+  margin: 0 auto;
+}
+
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-weight: bold;
+}
+
+h2 {
+  font-size: 16px;
+}
+
+h3 {
+  font-size: 24px;
+}
+
+h4 {
+  font-size: 20px;
+}
+
+h5 {
+  font-size: 17px;
+}
+
+h6 {
+  font-size: 19px;
+}
+
+p {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 15px;
+}
+
+.hashtag {
+  bottom: 0;
+  left: 0;
+}
+
+.secondary-hashtag {
+  bottom: 0;
+  left: 0;
+  background-color: #c2d4d7;
+  font-size: 13px;
+}
+
+.button-primary {
+  font-size: 18px;
+  font-weight: bold;
+}
+
+.button-secondary {
+  font-size: 18px;
+  font-weight: bold;
+}
+
+.text-main {
+  color: #f72210;
+}
+
+.bg-main {
+  background-color: #f72210;
+}
+
+.logotype-small {
+  width: 29px;
+  height: 28px;
+}
+
+.header-top {
+  background-color: #f72210;
+}
+
+.header-top-left img {
+  height: 20px;
+}
+
+.header-logo {
+  display: block;
+  left: 50%;
+  top: 25%;
+  margin-left: -200px;
+}
+
+.header-logo img {
+  height: 60px;
+}
+
+.header-top-right i {
+  font-size: 20px;
+  color: #fb9a94;
+}
+
+header .nav a {
+  font-size: 19px;
+  color: #000;
+}
+
+header .nav a::after {
+  content: "";
+  height: 100%;
+  padding-right: 1px;
+  margin: 0 6px;
+  background-color: #e2e2e2;
+}
+
+header input {
+  background-image: url("./imgs/search-icon.png");
+  background-repeat: no-repeat;
+  background-position: right;
+  background-size: 20px;
```

Comment:
- [ ] You could use a [background shorthand](https://developer.mozilla.org/en/docs/Web/CSS/background)